documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,200 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega - Creating Your First Application
+
+Learn how to create a Hello World Pravega app. This guide covers:
+
+* Starting Pravega standalone.
+* Creating a Pravega Stream.
+* Creating an Event Writer and write events into a Pravega Stream.
+* Create an Event Reader and read events from the Pravega Stream.
+
+ 
+# 1. Prerequisites
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 11+ installed with JAVA_HOME configured appropriately
+* Gradle 6.5.1+
+
+<details open>
+  <summary>Gradle 6.5.1+</summary>
+  Installation : https://gradle.org/install/
+<p>
+  !! Verify Gradle is using the Java you expect. You can verify which JDK Gradle uses by running `gradle --version`. !!
+</details>
+
+# 2. Goal
+In this guide, we will create a straightforward application that creates a Stream on Pravega and writes an event into the Stream and reads back from it.
+We recommend that you follow the instructions from [Bootstrapping project](#4-Bootstrapping-the-Project) onwards to create the application step by step.
+However, you can go straight to the completed example at [Pravega-samples-repo](https://github.com/pravega/pravega-samples).
+
+<details open>
+<summary>Command to clone the pravega-samples repo</summary>
+<p>
+
+```console
+$ git clone https://github.com/pravega/pravega-samples.git
+```
+
+</p>
+</details>  
+
+
+The solution is located in the pravega-client-examples [directory]( https://github.com/pravega/pravega-samples/tree/master/pravega-client-examples/src/main/java/io/pravega/example/gettingstarted ).
+
+# 3. Starting Pravega Standalone.
+In standalone mode, the Pravega server is accessible from clients through the `localhost` interface only. Controller REST APIs, however, are accessible from remote hosts/machines.
+You can launch a standalone mode server using either of the following options:
+
+1. From source code
```

Comment:
This enumeration sentence does not finish in `:`,  whereas the next one does. Again, let's keep consistency.

Reply:
fixed it.